client/db.go

```diff
@@ -74,10 +74,8 @@ func (kv *KeyValue) setValue(v *proto.Value) {
 	} else if v.Integer != nil {
 		kv.Value = v.Integer
 	}
-	if ts := v.Timestamp; ts != nil {
-		sec := ts.WallTime / 1e9
-		nsec := ts.WallTime % 1e9
-		kv.Timestamp = time.Unix(sec, nsec)
+	if v.Timestamp != nil {
```

Comment:
You mean the `ts := v.Timestamp`? That didn't seem as useful once most of the work was moved to `GoTime()`.